staging/istio/patch/patches/3_chartyaml.patch

```diff
@@ -5,25 +5,21 @@ Date:   Mon Apr 3 17:59:32 2023 -0700
     refactor: Update chart.yaml
 
 diff --git a/staging/istio/Chart.yaml b/staging/istio/Chart.yaml
-index cb1613b6dc..7f05e5acce 100644
+index 5ec8bd5265..b91793f481 100644
 --- a/staging/istio/Chart.yaml
 +++ b/staging/istio/Chart.yaml
-@@ -1,10 +1,7 @@
--apiVersion: v1
+@@ -1,7 +1,5 @@
```

Comment:
I had to update this patch because the upstream Chart.yaml https://github.com/istio/istio/blob/1.23.2/manifests/charts/istio-operator/Chart.yaml also updated since the last time we ran the upgrade script, so the diff/patch here would have changed. What I did was check out the upstream chart (I forked it) and made the changes we'd want for our chart fork, then copied over the new diff.